content/nginx-one/api/nginx_configuration.md

```diff
@@ -0,0 +1,128 @@
+    The following example demonstrates how to update the NGINX Config for a specific Instance using `PUT`. Replace `<tenant>`, `<namespace>`, `<instance-object-id>`, and `<token-value>` with your actual values. The request body should contain the complete NGINX Config in JSON format.
+    ```shell
+    curl -X PUT "https://<tenant>.console.ves.volterra.io/api/nginx/one/namespaces/<namespace>/instances/<instance-object-id>/config" \
+    -H "Authorization : APIToken <token-value>" \
+    -H "Content-Type: application/json" \
+    -d @updated_config.json
+    ```
+    - `<tenant>`: Your tenant name for organization plans.
+    - `<namespace>`: The namespace your Instance belongs to.
+    - `<instance-object-id>`: The object_id of the NGINX Instance you want to update the NGINX Config for.
+    - `<token-value>`: Your API Token.
+   
+2. **Update the NGINX Config for an Instance using `PATCH`**:
+
+    When using the `PATCH` method, you only need to include the files you want to update in your request body.
+    The following example demonstrates how to update the NGINX Config for a specific Instance using `PATCH`. Replace `<tenant>`, `<namespace>`, `<instance-object-id>`, and `<token-value>` with your actual values. The request body should contain only the fields you want to update in JSON format.
+    ```shell
+    curl -X PATCH "https://<tenant>.console.ves.volterra.io/api/nginx/one/namespaces/<namespace>/instances/<instance-object-id>/config" \
+    -H "Authorization : APIToken <token-value>" \
+    -H "Content-Type: application/json" \
+    -d @partial_update_config.json
+    ```
+    - `<tenant>`: Your tenant name for organization plans.
+    - `<namespace>`: The namespace your Instance belongs to.
+    - `<instance-object-id>`: The object_id of the NGINX Instance you want to update the NGINX Config for.
+    - `<token-value>`: Your API Token.
+
+    With `PATCH`, you can update specific parts of the NGINX Config without needing to resend the entire configuration. The following file `contents` disposition is observed:
+    - Leave out file `contents` to remove the file from the NGINX Config.
+    - Include file `contents` to add or update the file in the NGINX Config. File `contents` must be base64 encoded. File `contents` can be an empty string to create an empty file.
+    - `config_version` should be included to ensure you're updating the correct version of the configuration. You can get the current `config_version` from the response of the `GET` request.
+    For example, to update only the `nginx.conf` file in the NGINX Config, your `partial_update_config.json` might look like this:
+    ```json
+    {
+        "conf_path": "/etc/nginx/nginx.conf",
+        "config_version": "<config version from GET response>",
+        "configs": {
+            "/etc/nginx": {
+                "files": [{
+                    "path": "nginx.conf",
```

Comment:
based on API spec, I think this is `name` not `path`.

can we validate these instructions work once?